apps/web/components/I18nLanguageHandler.tsx

```diff
@@ -1,4 +1,4 @@
-import parser from "accept-language-parser";
+import { lookup } from "bcp-47-match";
```

Comment:
It's just 645 bytes after tree shaking @hariombalhara

Reply:
Awesome!!